app/main.py

```diff
@@ -1,3 +1,74 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float]) -> "Distance":
```

Comment:
Please, fix the type annotation here and in the other places to use the most recent approach 
```suggestion
    def __add__(self, other: Distance | float | int) -> Distance:
```
In case you face errors, use `from __future__ import annotations` import